symfony/test/functional/cypress/support/commands.js

```diff
@@ -43,3 +40,11 @@ Cypress.Commands.add(
     cy.get('form').submit();
   },
 );
+
+Cypress.Commands.add('truncateTable', (tables = []) => {
```

Comment:
Need proper function name rather than `truncateTable`. Seems it's not good name in Cypress test scripts